vendor/pear-pear.php.net/Text_LanguageDetect/Text/LanguageDetect.php

```diff
@@ -90,7 +90,7 @@ class Text_LanguageDetect
      *
      * @var string
      */
-    protected $_data_dir = '@data_dir@';
+    protected $_data_dir = 'D:\Mes Projets\Friendica\friendica\vendor/pear-pear.php.net/Text_LanguageDetect/data';
```

Comment:
Does it work when you change that? If yes, can you provide a PR?

Reply:
@data_dir@ is replaced at install by PEAR by the absolute path of the local install, but it has a relative path fallback in case the replacement didn't occur. I'll revert this hunk.